components/suggest/src/schema.rs

```diff
@@ -21,10 +21,10 @@ pub const SQL: &str = "
         PRIMARY KEY (keyword, suggestion_id)
     ) WITHOUT ROWID;
 
-    CREATE TABLE pocket_keywords( 
+    CREATE TABLE prefix_keywords(
         keyword_prefix TEXT NOT NULL,
         keyword_suffix TEXT NOT NULL DEFAULT '',
-        confidence INTEGER NOT NULL,
+        confidence INTEGER NOT NULL DEFAULT '',
```

Comment:
```suggestion
        confidence INTEGER NOT NULL DEFAULT 0,
```

`''` is technically legal—SQLite's type system is [relaxed by default](https://www.sqlite.org/datatype3.html)—but let's use a number if the column is declared as a number.